storage_service/locations/models/package.py

```diff
@@ -163,6 +163,28 @@ def is_compressed(self):
                 message = "{} is neither a file nor a directory".format(full_path)
             raise StorageException(message)
 
+    @property
+    def latest_fixity_check_datetime(self):
+        latest_check = self._latest_fixity_check()
+        if latest_check is not None:
+            return latest_check.datetime_reported
+        else:
+            return None
+
+    @property
+    def latest_fixity_check_result(self):
+        latest_check = self._latest_fixity_check()
+        if latest_check is not None:
+            return latest_check.success
+        else:
+            return None
+
+    def _latest_fixity_check(self):
+        try:
+            return FixityLog.objects.filter(package=self).order_by('-datetime_reported')[0]  # limit 1
```

Comment:
Please run a linter over your code before you commit it. It will find things like `FixityLog` being undefined.